packages/astro/e2e/fixtures/actions-blog/src/components/PostComment.tsx

```diff
@@ -17,7 +18,7 @@ export function PostComment({
 			<form
 				method="POST"
 				data-testid="client"
-				action={actions.blog.comment}
+				action={actions.blog.comment.toString()}
```

Comment:
Just noting that if this is required, it's a user-side bug. Understand if this is just a debug in-progress

Reply:
That's intentional. For some reason, the `toString()` isn't called anymore. Has something changed in the TSX compiler? I bet that if we use a template literal, the `toString()` function is called, however... like this, `toString()` isn't automatically called, so I am afraid the transformation is doing something fishy